t/http-cookiejar.t

```diff
@@ -92,6 +93,38 @@ $jar.add-from-response($resp, Cro::Uri.parse: 'http://example.com');
 my $req = Cro::HTTP::Request.new(method => 'GET', target => '/');
 $jar.add-to-request($req, Cro::Uri.parse: 'http://example.com/');
 like $req.Str, /Cookie/,  'Header was added';
-like $req.Str, /'Foo=Bar; Bar=Baz' || 'Bar=Baz; Foo=Bar'/,  'Setted string is correct';
+like $req.Str, /'Foo=Bar; Bar=Baz' || 'Bar=Baz; Foo=Bar'/,  'Set string is correct';
+
+$jar.clear;
+$jar.add-cookie(
+  Cro::HTTP::Cookie.new(
+    name    => 'Test',
+    value   => 'Cookie',
+    max-age => Duration.new(3600e0),
+    domain  => 'example.com',
+    path    => '/'
+  )
+);
+ok $jar.contents.elems == 1,                        'A single cookie was added, successfully';
```

Comment:
These tests would be better written like:

```
is $jar.contents.elems, 1, 'A single cookie was added, successfully';
```

Otherwise, should they fail, we don't get the got/expected diagnostic output.